Dockerfile

```diff
@@ -1,8 +1,57 @@
-FROM geonode/django
-MAINTAINER GeoNode development team
+FROM python:2.7.9
```

Comment:
The main reason here is that geonode can be easily customizable with a geonode-project which is a separated django application on top of that.

Can you describe in detail what the complications would be?